include/swift/AST/DiagnosticsSema.def

```diff
@@ -1535,6 +1535,12 @@ ERROR(types_not_equal_decl,none,
 ERROR(types_not_equal_in_decl_ref,none,
       "referencing %0 %1 on %2 requires the types %3 and %4 be equivalent",
       (DescriptiveDeclKind, DeclName, Type, Type, Type))
+ERROR(types_not_inherited_decl,none,
+      "%0 %1 requires that %2 inherit from %3",
```

Comment:
Should "inherit" be "inherits" or "is inherited"?

Reply:
We already had a diagnostic which said "inherit" which seemed appropriate.

I'm not a language expert (of any language), but it seems like zero infinitive form is valid here because "s/-es" is only used for single person singular present form?

Reply:
/conjure @jrose-apple

Reply:
"method 'blah()' requires that 'Child' inherit from 'Parent'". Pavel's way is correct.

Reply:
Thanks, @jrose-apple!